src/niweb/apps/userprofile/models.py

```diff
@@ -23,5 +25,24 @@ def url(self):
 
 @receiver(post_save, sender=User)
 def create_user_profile(sender, **kwargs):
+    import logging
```

Comment:
Double imports (also imports go at the top of files, except for rare "mess with paths stuff)